blosc/frame.c

```diff
@@ -438,6 +438,9 @@ int get_header_info(blosc2_frame *frame, int32_t *header_len, int64_t *frame_len
     // We can compute the number of chunks only when the frame has actual data
     *nchunks = (int32_t) (*nbytes / *chunksize);
     if (*nbytes % *chunksize > 0) {
+      if (*nchunks == INT32_MAX) {
```

Comment:
I think it would be safer and clearer to move this condition up by 1 line and add ` || *nchunks < 0`  to it.

Reply:
In that case max number of chunks would be `INT32_MAX - 1` because `*nchunks == INT32_MAX` would always return -1. Are you okay with that?

Reply:
Well spotted!